lib/IDE/CodeCompletion.cpp

```diff
@@ -3990,7 +4074,7 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
       builder.setLiteralKind(kind);
 
       consumer(builder);
-      addTypeRelationFromProtocol(builder, kind, defaultTypeName);
+      addTypeRelationFromProtocol(builder, kind);
```

Comment:
`defaultTypeName` is now unused, so you can remove it from the lambda and the callsites.  This applies to both the `addFromProto` lambdas in this patch.